core/slice.go

```diff
@@ -503,6 +503,9 @@ func (sl *Slice) GetManifest(blockHash common.Hash) (types.BlockManifest, error)
 // produced this block
 func (sl *Slice) GetSubManifest(slice common.Location, blockHash common.Hash) (types.BlockManifest, error) {
 	subIdx := slice.SubIndex()
+	if sl.subClients[subIdx] == nil {
+		return nil, errors.New("missing requested subordinate node")
```

Comment:
What if you are running a slice. wouldn't you error ? and if so, idk if that is desirable

Reply:
If you don't have the subordinate in question, then it is not possible to ask that subordinate for his manifest. That is an error case.

This should never happen, but one way it could happen is if a miner attempts to submit a mined block for the wrong slice. `ReceiveMinedHeader()` will attempt to reconstruct the block body with the subordinate manifest, but if that subordinate does not exist, then it should throw an error and fail to reconstruct the block.

Previously, if the miner sent a header for the wrong slice, the `ReceiveMinedHeader()` API would panic and crash the entire node.